Add illumination correction module for Plate 2 (#22)
WayScience/pyroptosis_signature: Plate_2_data/2.cellprofiler_ic_processing/README.md

# Create and edit LoadData csv and calculate illumination correction function

> In this module, we created a LoadData csv, removed rows except for those with the last z-plane and corrected paths to maximum projection images (as they have the same name as the last z-plane). We then ran a CellProfiler pipeline for calculating an illumination correction (IC) function (`.npy` file) for all images in each channel. 

Author: I tried making it clearer, but it is a bit hard to convey in writing. See https://github.com/WayScience/pyroptosis_signature/pull/22/commits/5798f7f15488298ce7a7fd5f745d9277e2e63f94

As stated in the [previous module README](Plate_2_data/1.cellprofiler_quality_control/README.md), this methodology is inspired by a project from the Broad Institute and we had the guidance of [Erin Weisbart](https://github.com/ErinWeisbart).

## Step 4: Create LoadData CSV, edit it, and calculate IC function for each channel in CellProfiler

> To create and edit the LoadData CSV and calculate the illumination correction function for both channels in the data, run the [loaddata_edit_and_ic.ipynb](loaddata_edit_and_ic.ipynb) note using the code below.

Reviewer: ```suggestion
To create, edit, and calculate illumination correction for the LoadData CSV data on both channels, run the [loaddata_edit_and_ic.ipynb](loaddata_edit_and_ic.ipynb) note using the code below.
```

Attempted to simplify. Feel free to reject this suggestion if I failed to convey your message.

Author: I like this! I have added this and also corrected the info in the sentence that was incorrect. See https://github.com/WayScience/pyroptosis_signature/pull/22/commits/5798f7f15488298ce7a7fd5f745d9277e2e63f94
